### PR TITLE
Remove Client's intervals and timeouts

### PR DESCRIPTION
**Pull request by SlimyBoy2**
It is the same thing as [this pr](https://github.com/discord-akairo/discord-akairo/pull/228/files) with the unref method as proposed

### CODE REVIEW

File: src/struct/commands/CommandHandler.js [hunk 1]
```diff
@@ -722,16 +722,16 @@ class CommandHandler extends AkairoHandler {
...
-                        this.client.clearTimeout(this.cooldowns.get(id)[command.id].timer);
+                        clearTimeout(this.cooldowns.get(id)[command.id].timer);
                     }
                     this.cooldowns.get(id)[command.id] = null;
 
                     if (!Object.keys(this.cooldowns.get(id)).length) {
                         this.cooldowns.delete(id);
                     }
-                }, time),
+                }, time).unref(),
```
Comment (almeidx) [inline src/struct/commands/CommandHandler.js:734]: Not sure if using the unref method here would be a great idea

As per the [Node.js docs](https://nodejs.org/api/timers.html#timers_timeout_unref)
> Calling `timeout.unref()` creates an internal timer that will wake the Node.js event loop. Creating too many of these can adversely impact performance of the Node.js application.

Comment (SlimyBoy2) [reply src/struct/commands/CommandHandler.js]: Ok yeah I didn't understand that. This part confused me. I guess they ment calling it multiple times on the same timeout.
> Calling `timeout.unref()` multiple times will have no effect.

I'll removed it

File: src/struct/commands/CommandHandler.js [hunk 2]
```diff
@@ -124,7 +124,7 @@ class CommandHandler extends AkairoHandler {
          */
         this.commandUtilSweepInterval = commandUtilSweepInterval;
         if (this.commandUtilSweepInterval > 0) {
-            this.client.setInterval(() => this.sweepCommandUtil(), this.commandUtilSweepInterval);
+            setInterval(() => this.sweepCommandUtil(), this.commandUtilSweepInterval).unref();
```
Comment (GamingGeek) [inline src/struct/commands/CommandHandler.js:127]:
```suggestion
            setInterval(() => this.sweepCommandUtil(), this.commandUtilSweepInterval);
```

Comment (Zordlan) [reply src/struct/commands/CommandHandler.js:127]: yooo its geek

**Pull request merged**